Sources/ParseSwift/API/URLSession+async.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  URLSession+async.swift
+//  ParseSwift
+//
+//  Created by Corey Baker on 7/22/20.
+//  Copyright © 2020 Parse. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+extension URLSession {
+
+    internal func asyncDataTask(with request: URLRequest,
+                                completion: @escaping(Result<Data, ParseError>) -> Void) {
+
+        let semaphore = DispatchSemaphore(value: 0)
```

Comment:
@mtrezza This should be commented out along with the rest of semaphore

Reply:
And what is the issue when you comment it out?

Reply:
The data will never be retrieved from the pare-server. In the previous PR, I added URLMocking, running a test with it shows no data is ever retrieved